components/micro-gateway-core/src/main/ballerina/gateway/executiontime_util.bal

```diff
@@ -0,0 +1,62 @@
+import ballerina/log;
```

Comment:
License header is missing. Please check all the files in the PR.

Reply:
Added